validator/keymanager/derived/mnemonic.go

```diff
@@ -98,6 +107,9 @@ func setBip39Lang(lang string) {
 
 	if wl, ok := allowedLanguages[lang]; ok {
 		wordlist = wl
+	} else {
+		return ErrUnsupportedMnemonicLanguage
```

Comment:
Would be nice to also include the lang in the error, something like errors.Wrapf(ErrUnsupportedMnemonicLanguage, "%s", lang)

Reply:
thank you. That's a great idea, adding it now.